### PR TITLE
Added necesary API endpoint

### PR DESCRIPTION
**Pull request by dane-johnson**
Necessary endpoints for ui issue https://github.com/maryvilledev/skilldirectoryui/issues/75.

### CODE REVIEW

File: controller/skillreviewscontroller.go [hunk 1]
```diff
@@ -83,6 +87,45 @@ func (c *SkillReviewsController) getAllSkillReviews() error {
 	return err
 }
 
+func (c *SkillReviewsController) getSkillReviewForSkill(skill_id string) error {
```
Comment (Hopding) [inline controller/skillreviewscontroller.go]: The name of this method confused me initially. I think changing the name to `getReviewsForSkill` (or `getSkillReviewsForSkill`) would be more clear.

**Review by Hopding — APPROVED** [review]
Awesome 👍

**Comment by dane-johnson** [comment]
This one will need to be merged to have the reviews render properly.

File: controller/skillreviewscontroller.go [hunk 2]
```diff
@@ -83,6 +87,45 @@ func (c *SkillReviewsController) getAllSkillReviews() error {
 	return err
 }
 
+func (c *SkillReviewsController) getReviewForSkill(skill_id string) error {
+	opts := data.NewCassandraQueryOptions("skill_id", "'" + skill_id + "'", true)
```
Comment (thebho) [inline controller/skillreviewscontroller.go]: Try this with false as the Id option.  If that doesn't work, I'll put this through and make a story about handling this kind of request

**Review by thebho — CHANGES_REQUESTED** [review]

**Review by thebho — APPROVED** [review]

**Pull request merged**